include/zephyr/arch/arm64/scripts/linker.ld

```diff
@@ -164,6 +164,14 @@ SECTIONS
         *(".rodata.*")
         *(.gnu.linkonce.r.*)
 
+        /*
+         * The following is a workaround to allow compiling with GCC 12 and
+         * above, which may emit "GOT indirections" for the weak symbol
+         * references.
+         */
```

Comment:
Can you add here a pointer to the discussion about this bug?

Reply:
thought it would be easily traceable from git blame, but would not hurt to add one here as well. done.